packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/updateJobLogForId..st

```diff
@@ -7,8 +7,11 @@ updateJobLogForId: aNumber
 		on: GHAAccessDeniedError 
 		do: [ :error |  response := 'Acces denied. Logs are only available when providing a token' ].
 		
-	(self isSmalltalkCILog: response)
-		ifTrue: [response := self parseSmalltalkCILog: response].
+	(self shouldSimplifySmalltalkCiLog)
+		ifTrue: [
+			(self isSmalltalkCILog: response)
+				ifTrue: [response := self parseSmalltalkCILog: response].
+		].
```

Comment:
```suggestion
	(self shouldSimplifyLog)
		ifTrue: [
			(self isSmalltalkCILog: response)
				ifTrue: [response := self parseSmalltalkCILog: response].
		].
```
I felt the names were a little too long to support readability. Although the parsing is in the same class i would not include any info on implementation (SmalltalkCiLog vs Log) in GUI code